src/IntegrationLoader.ts

```diff
@@ -67,16 +62,24 @@ export default class IntegrationLoader {
         case IntegrationName.FINGERPRINTJS:
           integrationClass = FingerprintJSIntegration;
           break;
+
         case IntegrationName.SIFT:
         case IntegrationName.SIFT_SCORE:
           integrationClass = SiftIntegration;
           break;
+
         case IntegrationName.STRIPE_IDENTITY:
           integrationClass = StripeIdentityIntegration;
           break;
+
         case IntegrationName.MFA:
           integrationClass = MfaIntegration;
           break;
+
+        case IntegrationName.KOUNT:
+          integrationClass = libConfig?.config["global.browserTracking"] === "true" ? KountIntegration : null;
```

Comment:
This logic should live on the API side. We should not send over the libConfig for Kount if the browserTracking flag is not true.

Reply:
Makes sense. I'll refactor and update both PRs. Thanks!